assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -586,15 +599,44 @@ subroutine update_inflation(inflate_handle, inflate, inflate_sd, prior_mean, pri
 ! select which method to update with
 if (do_enhanced_ss_inflate(inflate_handle)) then
    inf_type = GHA2017
+   test_flavor = 5;
```

Comment:
do these need parameters, or are they the same values as the inf_type parameters?  i hate to see more magic numbers added.